nnpdf_data/nnpdf_data/commondata/D0_WPWM_1P96TEV/metadata.yaml

```diff
@@ -9,6 +9,7 @@ arXiv:
   journal: Phys. Rev. D91 (2015) 032007
 iNSPIRE:
   url: https://inspirehep.net/literature/1253555
+  version: 1
```

Comment:
I mean the `version_comment` in the top of the file, but I could not leave the comment there.